src/runtime/components/prose/Accordion.vue

```diff
@@ -36,7 +38,7 @@ const rerenderCount = ref(1)
 const items = computed<{
   index: number
   label: string
-  icon: string
+  icon: Exclude<IconProps['name'], Component>
```

Comment:
I'd just keep a string here:
```suggestion
  icon: string
```